flutter/lib/src/default_integrations.dart

```diff
@@ -47,7 +47,7 @@ Integration loadContextsIntegration(
 ) {
   Future<void> integration(Hub hub, SentryOptions options) async {
     options.addEventProcessor(
-      (event, dynamic hint) async {
+      (event, {hint}) async {
```

Comment:
```suggestion
      (event, {dynamic hint}) async {
```
we should avoid implicit dynamic

Reply:
wondering if the `event` should be typed too then?